app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/actions.ts

```diff
@@ -59,3 +61,44 @@ export const unConfirmedResponsesExist = AuthenticatedAction(async (_, formId: s
     return { error: "There was an error. Please try again later." } as ServerActionError;
   }
 });
+
+export const getEventsForForm = AuthenticatedAction(async (_, formId: string) => {
+  try {
```

Comment:
I think we need to add a permission check here at the beginning to make sure the user has access to the requested form ID. 

Not sure if we leverage `canViewForm` or `canEditForm` though 🤔 

Probably even better to include it directly in the lib layer where we actually make the calls to DynamoDB to get audit logs (like we do with the templates lib file) so that any call to the function from any places in the code is safe by default